src/Sulu/Bundle/WebsiteBundle/EventListener/AppendAnalyticsListener.php

```diff
@@ -59,7 +59,7 @@ public function __construct(
      */
     public function onResponse(FilterResponseEvent $event)
     {
-        if ($event->getRequest()->getRequestFormat() !== 'html'
+        if (0 !== strpos($event->getResponse()->headers->get('Content-Type'), 'text/html')
```

Comment:
the format is not set for example in a image request. but the mimetype of the response is always correct and can be used for this statement.